standalone/include/psen_scan_v2_standalone/data_conversion_layer/io_constants.h

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#ifndef PSEN_SCAN_V2_STANDALONE_IO_CONSTANTS_H
+#define PSEN_SCAN_V2_STANDALONE_IO_CONSTANTS_H
+
+#include <array>
+#include <map>
+#include <string>
+
+namespace psen_scan_v2_standalone
+{
+namespace data_conversion_layer
+{
+namespace monitoring_frame
+{
+namespace io
+{
+/**
+ * @brief Contains constants and types needed to define PinData and for the (de)serialization.
+ */
+static constexpr uint32_t RAW_CHUNK_LENGTH_RESERVED_IN_BYTES{ 4 };
+static constexpr uint32_t RAW_CHUNK_PHYSICAL_INPUT_SIGNALS_IN_BYTES{ 10 };
+static constexpr uint32_t RAW_CHUNK_LOGICAL_INPUT_SIGNALS_IN_BYTES{ 8 };
+static constexpr uint32_t RAW_CHUNK_OUTPUT_SIGNALS_IN_BYTES{ 4 };
+static constexpr uint32_t RAW_CHUNK_LENGTH_IN_BYTES{
+  3 * (RAW_CHUNK_LENGTH_RESERVED_IN_BYTES + RAW_CHUNK_PHYSICAL_INPUT_SIGNALS_IN_BYTES) +
+  RAW_CHUNK_LENGTH_RESERVED_IN_BYTES + RAW_CHUNK_LOGICAL_INPUT_SIGNALS_IN_BYTES + RAW_CHUNK_LENGTH_RESERVED_IN_BYTES +
+  RAW_CHUNK_OUTPUT_SIGNALS_IN_BYTES
+};
+
+using RawChunk = std::array<uint8_t, io::RAW_CHUNK_LENGTH_IN_BYTES>;
+
+#define REV(arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8) arg8, arg7, arg6, arg5, arg4, arg3, arg2, arg1
```

Comment:
Duplication

Reply:
I would keep it instead of creating a new file just for this define.